### ISSUE TITLE
Definition of CSV file

### ISSUE THREAD

**Issue by chrbertsch** [PMSFIT/fmi-ls-ref#4]
Do we need to reference a definition of the CSV file?

Currently we have 

- https://github.com/modelica/fmi-standard.org/blob/main/CONTRIBUTING.md#csv-rules-for-inout-and-result-files-for-compatibility-information
- https://github.com/modelica/Reference-FMUs?tab=readme-ov-file#csv-file-structure

**Comment by pmai** [comment]
Yes, the idea would be to reference a definition once it is clear where the official one will be located (could also be transferred into LS-REF, if we have no other place that's purely specification and not implementation/use).

**Comment by chrbertsch** [comment]
@pmai workson a common definition for MA Standards, see proposal: https://github.com/modelica/MAP-COORD/blob/main/Technical/CSV/MA-CSV.md
